simulators/nav/src/store/modules/roverState.ts

```diff
@@ -50,7 +50,8 @@ const state:RoverState = {
 
   obstacleMessage: {
     distance: -1,
-    bearing: 0
+    bearing: 0,
+    size: 0.5
```

Comment:
what is this the size of?

Reply:
The size of an obstacle. It is for calculating its bounding box